README.md

```diff
@@ -1,17 +1,141 @@
-# Project
+# GitHub Action for deploying to Azure Spring Cloud
+GitHub Actions support an automated software development lifecycle workflow. With GitHub Actions for Azure Spring Cloud you can create workflows in your repository to manage your deployment of Azure Spring Cloud conveniently.
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[comment]: <> (This repository contains GitHub Action for Azure Spring Cloud to deploy. The action supports deploying *.jar, *.war, and *.zip files, or a folder.)
```

Comment:
is `*.war` supported?

Reply:
Sorry, latest version not saved, comments need to be deleted.